packages/api-bindings/src/graphql/utils/publication.ts

```diff
@@ -65,24 +65,37 @@ export function getApiReactionType(reaction: ReactionType): ReactionTypes {
   }
 }
 
-export type PublicationFragment = PostFragment | CommentFragment | MirrorFragment;
+export type AnyPublicationFragment = CommentFragment | MirrorFragment | PostFragment;
+
+export type ContentPublicationFragment = CommentFragment | PostFragment;
```

Comment:
Once we will get to remove `Fragment` these types will look pretty self-explanatory in hooks responses/inputs.

Reply:
yes, agree